server/cluster/cluster.go

```diff
@@ -1650,6 +1654,66 @@ func (c *RaftCluster) RemoveStoreLimit(storeID uint64) {
 	log.Error("persist store limit meet error", errs.ZapError(err))
 }
 
+// GetMinResolvedTS returns the min resolved ts of all stores.
+func (c *RaftCluster) GetMinResolvedTS() uint64 {
+	c.RLock()
+	defer c.RUnlock()
+	if !c.isInitialized() {
+		return math.MaxUint64
+	}
+	min := uint64(math.MaxUint64)
+	for _, s := range c.GetStores() {
+		if !core.IsAvailableForMinResolvedTS(s) {
```

Comment:
Do we need to set a ttl for min resolved-ts reported? It will solve a store with leader but not report min resolved-ts for a long time @disksing

Reply:
If tikv has bug or wrong config and it does not report `min-resolved-ts`, I guess the global min-resolved-ts should not advance, or the result could be wrong.